### ISSUE TITLE
Evaluation failed: Error: MediaFileEmpty

### ISSUE THREAD

**Issue by arbisyarifudin** [orkestral/venom#1725]
## Description

Hi everyone! I got an error when upload a file. I attach the detail below:

## Environment

- **Venom version(s):** 4.1.0
- **Browser:** Chromium 
- **OS:** Windows 10
- **Node version:** Node 16

## Steps to Reproduce

1. Upload a Pdf file
2. Get file URL from server
3. Download URL to local 
4. Run SendFile with the local file path
5. Got Error

## Log Output

```
Error: Evaluation failed: Error: MediaFileEmpty
    at Object.sendToChat (https://web.whatsapp.com/bootstrap_qr.a817218351c7d3e5be74.js:71:336622)
    at Object.u [as sendFile] (<anonymous>:1:25599)
    at ExecutionContext._evaluateInternal (D:\Test\venom-express-api\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:221:19)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async ExecutionContext.evaluate (D:\Test\venom-express-api\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:16)
    at async D:\Test\venom-express-api\node_modules\venom-bot\dist\api\layers\sender.layer.js:528:28
```

## My Code

```
    let data = Sessions.getSession(session)
    let number = req.body.number + '@c.us'
    let isURL = await urlExists(req.body.path)
    let name = req.body.path.split(/[\/\\]/).pop()
    try {
      if (isURL) {
        await get(req.body.path, {
          directory: 'files-received',
          filename: name
        })
        let response = await data.client.sendFile(
          number,
          `./files-received/${name}`,
          'File-' + req.body.messageId,
          req.body.caption
        )
        console.log('response sendFile', response)
        fs.unlink(path.basename('/files-received') + '/' + name, (erro) =>
          console.log('')
        )
        
        return res.status(200).json({
          result: 200,
          type: 'file',
          messageId: response.id,
          session: session,
          file: name,
          data: response
        })
      }
      if (!isURL) {
        return res.status(400).json({
          result: 400,
          status: 'FAIL',
          log: 'URL not found!'
        })
      }
    } catch (error) {
      return res.status(400).json({
        result: 400,
        status: 'FAIL',
        log: error
      })
    }
```

**Comment by orkestral** [comment]
Fix #1719

**Comment by arbisyarifudin** [comment]
> Fix #1719

Hi @orkestral , it solves my first problem! Instead of error mediaFileEmpty and make the server down it gives the correct response file 400. 
But I got a problem when I upload file that hassize under 50 KB. It's always gives me fail message:
```
"No such file or directory, open \"files-received/FL62a2953e9b4f70.docx\""
```

But how? All the files I uploaded were successful. I got this error only when uploading a file whice size is under 50KB.
